mcda/utils/utils_for_main.py

```diff
@@ -996,17 +1055,18 @@ def _compute_scores_for_all_random_weights(indicators: dict, is_sensitivity: str
     args_for_parallel_agg = [(lst, indicators) for lst in random_weights]
 
     if is_sensitivity == "yes":
-        all_weights_scores = parallelize_aggregation(args_for_parallel_agg)  # rough scores coming from all runs
+        all_weights_scores = utils_for_parallelization.parallelize_aggregation(args_for_parallel_agg)
+        #  rough scores coming from all runs
```

Comment:
Now the comment looks like it is intended for `else`.

Reply:
Maybe don't remove the comment but move it on the top of the code line.

Reply:
Not needed really.